WordPress/Classes/Utility/Logging/WPCrashLogging.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+import Sentry
+
+fileprivate let UserOptedOutKey = "crashlytics_opt_out"
```

Comment:
Am I right in assuming that users who are currently opted out will remain opted out after the switch?

Reply:
Yep!